src/components/pages/PowerheadPage/model.js

```diff
@@ -67,7 +67,15 @@ function makeFilterByLocationFn$(selectedLocation$) {
         || location === report.country
         || location === report.site
       );
-      return {...oldState, reports: newReports};
+
+      return {
+        ...oldState,
+        reports: newReports,
+        filters: {
+          ...oldState.filters,
+          location: location,
+        },
+      };
```

Comment:
The filter updates the reports in the state, but before this PR it would not update the filter values. I'm checking in the [view](https://github.com/futurice/power-ui/blob/powerhead-page-show-all/src/components/pages/PowerheadPage/view.js#L246) wheter `location === 'all'` and create a combined report if true. So the view needs not only the filtered reports but also what filters were used to create the filtered list. I think it would make sense for other filters to add this data too.